### DISCUSSION TITLE
How to change user avatar link?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#17105] [Archive]
Instead of displaying their profile, I have to redirect the user to a custom page.
How can I set the href of `a.router-link-active.router-link-exact-active`?

**Comment by rijkvanzanten** [comment]
There's no way to configure that currently

**Reply by Pachat** [reply]
OK. Why not follow https://github.com/directus/directus/discussions/12266#discussioncomment-3808995?

**Comment by Pachat** [comment]
I ended creating a module that makes the wanted redirection.
